hometask_html_css/style.css

```diff
@@ -0,0 +1,163 @@
+body {
+    font-family: sans-serif;
+}
+
+/*
+Styling the header region
+*/
+header {
+    background-color: #060f48;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: 100px;
+}
+
+.header-content {
+    background-color: white;
+    text-align: center;
+    width: 40%;
+    height: 50px;
+    margin: 20px auto;
+    box-sizing: border-box;
+    border: 1px solid rgb(249, 174, 12);
+    font-weight: bolder;
+}
+
+.main-title {
+    font-size: 20px;
+    margin: 0;
+    padding: 10px 0 5 0px;
+}
+
+.sub-title {
+    font-size: 10px;
+    margin: 0;
+}
+
+/*
+Styling the category selection and subscribe section
+*/
+.category-container {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    height: 100px;
+    grid-column-gap: 50px;
+
+}
+
+.category-item {
+    align-self: center;
+    padding-left: 15px;
+}
+
+.category-container {
+    border-bottom: 1px solid black;
+}
+
+.allnews-headingitem {
+    justify-self: center;
+}
+
+.category {
+    margin-left: 20px;
+}
+
+.category select {
```

Comment:
you can give ID here else it will apply same css on every block present in category class